services/udp/service.go

```diff
@@ -18,10 +18,6 @@ import (
 )
 
 const (
-	// UDPBufferSize is the maximum UDP packet size
-	// see https://en.wikipedia.org/wiki/User_Datagram_Protocol#Packet_structure
-	UDPBufferSize = 65536
```

Comment:
There's nothing wrong with allocating a buffer size > 65536, the only reason that value was selected was to provide guidance on an upper bound. Nothing stopping you from setting that higher if you really want (in terms of usability).